src/modules/Colorbar/info.json

```diff
@@ -16,17 +16,60 @@
         "x": {
             "type": "integer",
             "desc": "X-position of the image on which the new image is overlayed",
-            "default": 0
+            "default": "450"
```

Comment:
Shouldn't it depend on the image. What if the dimension of image is less than 450

Reply:
This is just default input, use can change according to him